libcodechecker/log/log_parser.py

```diff
@@ -206,6 +206,21 @@ class ImplicitCompilerInfo(object):
     compiler_target = {}
     compiler_standard = {}
 
+    compiler_isexecutable = {}
+
+    def is_executable_compiler(self, compiler):
+        if compiler not in self.compiler_isexecutable:
+            proc = subprocess.Popen(['which', compiler],
```

Comment:
This is quite linux specific there are other methods to [find an executable](https://stackoverflow.com/questions/377017/test-if-executable-exists-in-python) please make a note for a python3 solution. Later we can upgrade to it. Using `distutils.spawn.find_executable` might be better in this case.